packages/venia-ui/lib/components/CheckoutPage/ShippingMethod/shippingMethod.gql.js

```diff
@@ -3,27 +3,19 @@ import gql from 'graphql-tag';
 import { PriceSummaryFragment } from '@magento/venia-ui/lib/components/CartPage/PriceSummary/priceSummaryFragments';
 
 import {
-    SelectedShippingMethodFragment,
-    ShippingMethodsFragment
+    AvailableShippingMethodsFragment,
+    SelectedShippingMethodFragment
 } from './shippingMethodFragments.gql';
 
-export const GET_SHIPPING_METHODS = gql`
-    query GetShippingMethods($cartId: String!) {
-        cart(cart_id: $cartId) @connection(key: "Cart") {
-            id
-            ...ShippingMethodsFragment
-        }
-    }
-    ${ShippingMethodsFragment}
-`;
-
-export const GET_SELECTED_SHIPPING_METHOD = gql`
-    query GetSelectedShippingMethod($cartId: String!) {
+export const GET_SELECTED_AND_AVAILABLE_SHIPPING_METHODS = gql`
```

Comment:
In most of the `gql.js` files in our codebase, we return an object of type

```es6
{
   queries: {},
   mutations: {}
}
```

Reply:
Resolved by https://github.com/magento/pwa-studio/pull/2382/commits/e94550de3971ae4f76375e9610986122e8c17afa.